public-images/index.rst

```diff
@@ -53,9 +53,9 @@ suggestions, fixes and constructive feedback.
    :hidden:
    :maxdepth: 1
 
+   public-images-explanation/index
```

Comment:
Our usual navigation sequence is how-to guides - explanation - reference. So could you move this line below the one for how-tos?